dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// The context to be provided by the choice behavior consumer in order to obtain the choice behavior configuration.
+/// </summary>
+[Experimental("SKEXP0001")]
+public class FunctionChoiceBehaviorContext
```

Comment:
Do we intent to have specializations of the context?

```suggestion
public sealed class FunctionChoiceBehaviorContext
```